src/utils/validations/validator/validator.spec.js

```diff
@@ -0,0 +1,127 @@
+import validator from '.';
+import PresenceValidator from '../presence';
+
+const shortErr = 'value is too short!';
+const presErr = 'this value is required!';
+const notZeroErr = 'this is zero!';
+const asyncErr = 'not foo!';
+const legacyErr = '[missing "en.errors.messages.blank" translation]';
+
+const runValidation = async (validations, value) => {
+  const validate = validator(validations);
+  let valid;
+  let index = 0;
+  await validate(value)
+    .then(() => {
+      valid = true;
+      index += 1;
+    })
+    .catch((err) => {
+      if (err.message) {
+        valid = err.message;
+      } else {
+        valid = Array.isArray(validations) ? validations[index].message() : validations.message();
```

Comment:
why do you need this if you have `handleLegacyValidation`?

Reply:
Long story short, we don't, I should of refactored this test when I changed the validator code 🤦‍♂️